arch/arm/boot/dts/nxp/imx/imx6ul-ts7250v3.dtsi

```diff
@@ -91,7 +81,7 @@
 
 	an_3p3v: an_3p3v {
 		compatible = "voltage-divider";
-		io-channels = <&supervisor_adc 0>;
+		io-channels = <&wizard_adc 0>;
```

Comment:
Changes from `supervisor` to `wizard` should have been fully covered in #225. If these were missed (it happens), they should be redone as a different PR, no need to pollute this PR with other changes.

This applies to multiple changes in this file (`arch/arm/boot/dts/nxp/imx/imx6ul-ts7250v3.dtsi`) as well as `arch/arm/boot/dts/nxp/imx/imx6ul-ts7250v3.dts`